### DISCUSSION TITLE
pls check it out mine it that right or wrong?

### DISCUSSION THREAD

**Discussion by SaiWaiYanMaungMaung** [codewars/codewars.com#2798] [General]
def type_validation(variable, _type):
    # your code here
    if type(variable)==type(_type):
     return True
     print( "{}{}".format(variable, _type))
    elif type(variable) != type(_type):
     return False
     print("{}{}".format(variable,_type))

g=type_validation("as",2)
print(g)

**Comment by hobovsky** [comment]
Its wrong.

Github discussions are meant to discuss things related to the Codewars website. If you  want to discuss or ask questions related to specific kata or code, join Codewars Discord server!

**Reply by SaiWaiYanMaungMaung** [reply]
ohh thanks for you remind  i didn't know about programming much cuz i just  newbie in this field btw can i get discord sever invite link ?if you don't mind

**Reply by Blind4Basics** [reply]
https://discord.com/invite/mSwJWRvkHA
